go.mod

```diff
@@ -4,9 +4,9 @@ go 1.18
 
 require (
 	github.com/actgardner/gogen-avro/v10 v10.2.1
-	github.com/aws/aws-sdk-go v1.44.81
-	github.com/aws/aws-sdk-go-v2/service/dynamodb v1.16.0
-	github.com/brave-intl/bat-go/libs v0.0.0-20220913154833-730f36b772de
```

Comment:
seems like this is moving to a prior version than what is in master?

Reply:
fixed